webui/react/src/components/CreateUserModal.tsx

```diff
@@ -140,10 +140,6 @@ const CreateUserModalComponent: React.FC<Props> = ({ onClose, user, viewOnly }:
     });
   }, [form, user, userRoles]);
 
-  if (user !== undefined && userRoles === null && rbacEnabled && canAssignRoles({})) {
-    return <Spinner tip="Loading roles..." />;
```

Comment:
this is rendered even if the modal is closed since this element is outside of `Modal`.